seeds/dex/aggregator/dex_aggregator_seed.csv

```diff
@@ -128,3 +128,6 @@ optimism,firebird_finance,1,2022-09-27,0x77266a3fb99476be6859436eb0000ec1d0a0a78
 polygon,bebop,jam,2023-12-26,0x2d557f42bc297c4a80953ab411b0215332a479f9c8965a94c35d1116a43e7405,361,"0,0",0x8f3cf7ad23cd3cadbd9735aff958023239c6a063,0.02490757551521891,0x2f6f07cdcf3588944bf4c42ac74ff24bf56e7590,0.19901417214943845
 arbitrum,bebop,jam,2023-12-22,0x9af150d67bf4a45f431eaea2b31c45b545caf142b4e407e66f84893bd644e5f7,15,"0,0",0x11cdb42b0eb46d95f990bedd4695a6e3fa034978,369.4936753177027,0xda10009cbd5d07dd0cecc66161fc93d7c9000da1,224.0498860609381
 ethereum,bebop,jam,2023-12-11,0x8487dba2850c0dc4e9760eaf96d1912e13a0c1dc5632dc82d1a8fa362e853cce,646,"0,0",0x6b175474e89094c44da98b954eedeac495271d0f,42.825602116243985,0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48,82.019954
+optimism,odos,1,2023-07-26,0x290383f03b313583642d6ba4e62152e59b1d950403978303848ea4b3a0032651,113,-1,0xdfa46478f9e5ea86d57387849598dbfb2e964b02,10664403029400947636838,0x7f5c764cbc14f9669b88837ca1490cca17c31607,10000000000
+optimism,odos,1,2023-07-26,0xf85d6e2c80fb384e0d25e756c8c24bca3c1e7620916c2d5eb508d4f4ab2d0a84,87,-1,0x7f5c764cbc14f9669b88837ca1490cca17c31607,50486338,0xda10009cbd5d07dd0cecc66161fc93d7c9000da1,50000000000000000000
+optimism,odos,1,2023-07-26,0xdcb73a07d46947eab2fd32ecd9b111c5aba8f421dea4a5d663f097959a7557ba,53,-1,0xdfa46478f9e5ea86d57387849598dbfb2e964b02,5326167530957657782642,0x7f5c764cbc14f9669b88837ca1490cca17c31607,5000000000
```

Comment:
`-1` in the seed may have mismatched type to trace_address.

Reply:
I think issue is not there, because after removing it also i get the same error.